src/test-integration/java/io/github/jopenlibs/vault/v1_11_4/api/LeasesTests.java

```diff
@@ -40,29 +41,96 @@ public class LeasesTests {
     @BeforeClass
     public static void setupClass() throws IOException, InterruptedException {
         container.initAndUnsealVault();
+        container.setupBackendDatabase(DbContainer.hostname);
     }
 
     @Before
     public void setup() throws VaultException {
         vault = container.getRootVault();
     }
 
-    @Test
+    @Test(expected = VaultException.class)
     public void testRevoke() throws VaultException {
-        final VaultResponse response = vault.leases().revoke("sys/revoke-prefix/dummy");
+        List<String> creationStatements = new ArrayList<>();
+        creationStatements.add(
+                "CREATE USER \"{{name}}\" WITH PASSWORD '{{password}}'; GRANT ALL PRIVILEGES ON DATABASE \"postgres\" to \"{{name}}\";");
+
+        DatabaseResponse databaseResponse = vault.database().createOrUpdateRole("new-role",
+                new DatabaseRoleOptions().dbName("postgres")
+                        .creationStatements(creationStatements));
+        TestCase.assertEquals(204, databaseResponse.getRestResponse().getStatus());
+
+        DatabaseResponse credsResponse = vault.database().creds("new-role");
+        TestCase.assertEquals(200, credsResponse.getRestResponse().getStatus());
+
+        TestCase.assertTrue(credsResponse.getCredential().getUsername().contains("new-role"));
```

Comment:
The code block look like duplicated at the tests below.
Is is make sense to add one into init method with @Before?

Reply:
Usually I prefer to write tests more verbose, to use them as "raw documentation". However, in this case verbosity is useless, because the setup is not directly concerned to scope of test. So, I've moved redundant code in dedicated method